Dockerfile

```diff
@@ -67,8 +68,7 @@ RUN echo '.libPaths(c("~/R/library",paste0(R.home(), "/library"), .libPaths()))'
 #
 # Install Conda Packages
 COPY files/conda_packages /tmp/
-RUN conda config --add channels bioconda \
-    && conda config --add channels conda-forge \
+RUN conda config --system --set pip_interop_enabled True \
```

Comment:
What is this doing? [I imagine this is](https://www.anaconda.com/conda-4-6-release/) why we're using `latest` version of conda then.

Reply:
Hopefully it will work well, it's kinda brand new :)

The `Speed` section of the release notes is interesting too. I'm not 100% if newer version's resolver is faster out-of-the-box or you have to use that `pycryptosat` to have decent speed 🤷‍♂

Reply:
From the wording it seems like the there is more work they're doing but it should be faster ⏩ 

> There’s more to do, but we hope you’ll notice things going faster.

Reply:
I'm not using the new solver but will give it a try later, but I did want the pip interop stuff so if someone has a library installed through pip, then conda won't try to install it again